### ISSUE TITLE
no-partially-commented-tag-lines doesn't work for features, examples and some scenario cases.

### ISSUE THREAD

**Issue by xgbuils** [vsiakka/gherkin-lint#153]
Hi!

It doesn't report an error but feature tags are partially commented:
``` gherkin
@featuretag1 #@featuretag2
Feature: Feature

Scenario: This is a Scenario
  Then this is a then step
```

It reports and error but scenario tags are not partially commented:
``` gherkin
Feature: Feature

@featuretag1 @featuretag2 # my comment
Scenario: This is a Scenario
  Then this is a then step
```

It doesn't report an error but example tags are partially commented:
``` gherkin
Feature: Feature

Scenario Outline: Scenario
  Then this is a then step <foo>
@tag1 # @tag2
Examples:
  | foo |
@tag3
Examples:
  | bar |
```

**Comment by xgbuils** [comment]
After reading #13 I understand what is the intention of this rule. Then, the second example I exposed is not valid. I'll fix first and third.
